lerobot/common/policies/act/modeling_act.py

```diff
@@ -101,7 +101,7 @@ def forward(self, batch: dict[str, Tensor]) -> dict[str, Tensor]:
             F.l1_loss(batch["action"], actions_hat, reduction="none") * ~batch["action_is_pad"].unsqueeze(-1)
         ).mean()
 
-        loss_dict = {"l1_loss": l1_loss}
+        loss_dict = {"l1_loss": l1_loss.item()}
```

Comment:
While it's verifying that in `act` there are no tensors/np arrays being logged, if we want to go ahead with the above change we should verify that no such values appear on other policies as well.

Reply:
I can confirm that the other policies conform.